Harvest.Web/Helpers/LogConfiguration.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+using Serilog.Events;
+using Serilog.Exceptions;
+
+namespace Harvest.Web.Helpers
+{
+    public static class LogConfiguration
+    {
+        private static bool _loggingSetup;
+
+        private static IConfiguration _configuration;
+
+        public static void Setup(IConfiguration configuration)
```

Comment:
@sprucely is all of this recommended or is this just copied from an old project?  On my newer projects I've left most of this out.  is it doing something important.

Reply:
Just about all examples I've seen use appsettings.json for Serilog config. But I copied from Peaks, since I thought it was the current/preferred way for our systems. I can go back and change it.

Reply:
oh I don't mean the appsettings stuff when I said is all of this recommended, I mean the stuff in this method, like
```            
AppDomain.CurrentDomain.UnhandledException += (sender, e) => Log.Fatal(e.ExceptionObject as Exception, e.ExceptionObject.ToString());

AppDomain.CurrentDomain.ProcessExit += (sender, e) => Log.CloseAndFlush();

#if DEBUG
            Serilog.Debugging.SelfLog.Enable(msg => Debug.WriteLine(msg));
#endif
            _loggingSetup = true;
```